src/main/java/mate/academy/service/UserService.java

```diff
@@ -1,20 +1,24 @@
 package mate.academy.service;
 
+import java.util.Arrays;
 import mate.academy.model.User;
 
 public class UserService {
-    private static final User[] users = new User[] {
+    private static final User[] users = new User[]{
```

Comment:
don't forget about whitespaces
```suggestion
    private static final User[] users = new User[] {
```